README.md

```diff
@@ -54,6 +54,9 @@ For prerequisites, installation and setup, see the [Installation and setup](docs
 ## Known issues
 See the [Known issues](docs/known-issues.md) documentation.
 
+## Troubleshooting
+Ran into problems? See the [troubleshooting](docs/content/troubleshooting.md) documentation.
```

Comment:
This might seem really picky, but I'd go for "running" instead of "ran" - it just sounds more natural.

Reply:
I like the actual text!

I'd go for "Run into problems?"

Run (present tense), ran (past simple), run (present perfect).

Present perfect implies a recent event. (More grammar facts available if required.)

Reply:
I was thinking about going for "run" as in the present perfect ("Have you run into problems?") but then thought it might be confused with "run" (present). So I went for "running" (continuously/repeatedly) instead 😁 

I don't really mind though. Run/running is fine, but ran looks a bit strange to me.

Reply:
Ran does look strange. I'm a programmer.